data/mods/Aftershock/maps/mapgen/stratoscomm_relay.json

```diff
@@ -0,0 +1,129 @@
+      "rows": [
+        "                        ",
+        "                        ",
+        "                        ",
+        "                   .    ",
+        "                    .   ",
+        "                   ..   ",
+        "                  |,,|  ",
+        "                 1|!!|  ",
+        "  p p p   -----111|55|  ",
+        "  p p p   -...----|66|  ",
+        "  p p p   -.4.!..>||||  ",
+        "  p p p  1-...----      ",
+        "         1--2--11       ",
+        "         11-#-11        ",
+        "         1              ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        "
+      ],
+      "palettes": [ "afs_habitat_structure", "afs_habitat_mech_furnishing" ],
+      "terrain": { "1": "t_dirtmound", "2": "t_bars", "5": "t_ramp_down_high", "6": "t_ramp_down_low" },
+      "furniture": { "4": "f_mil_comms_sat" }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "afs_stratoscomm_relay_s1" ],
+    "weight": 200,
+    "object": {
+      "fill_ter": "t_metal_floor",
+      "rows": [
+        "########################",
+        "########################",
+        "########################",
+        "########################",
+        "########################",
+        "########################",
+        "########################",
+        "########################",
+        "##################||||##",
+        "##########|||||||||55|##",
+        "##########|431=..<|66|##",
+        "##########|2.ö=!!||]]|##",
+        "##########|2h.!..|l..|##",
+        "##########|4,.=..|l..|##",
+        "##########||===..||]||##",
+        "#########|||yhv.....A|##",
+        "########||*|q.ö.hfh.P|##",
+        "########|~.+....hfh.c|##",
+        "########||&|+|+|...hk|##",
+        "#########|||@|@|FoyyE|##",
+        "###########|||||||||||##",
+        "########################",
+        "########################",
+        "########################"
+      ],
+      "palettes": [ "afs_habitat_structure", "afs_habitat_mech_furnishing" ],
+      "terrain": { "5": "t_ramp_up_high", "6": "t_ramp_up_low", "#": "t_rock" },
+      "furniture": {
+        "@": "f_sleep_pod",
+        "1": [ "f_mil_comms_power", "f_mil_comms_power_broken" ],
+        "2": "f_mil_comms_terminal",
+        "3": "f_mil_comms_heatsink",
+        "4": [ "f_mil_comms_process", "f_mil_comms_process_broken" ],
+        "o": "f_habitat_kitchenette",
+        "F": "f_fridge"
+      },
+      "items": {
+        "o": { "item": "afs_kitchenette", "chance": 100 },
+        "F": { "item": "afs_cheap_food", "chance": 80, "repeat": [ 1, 10 ] }
+      },
+      "monster": { "ö": { "monster": "mon_skitterbot" } }
```

Comment:
Am I missing the terrain that goes below `"ö"`.  I'm not seeing it set in the mapgen or the palettes?

Reply:
I'd missed the fill terrain sorry.